public/index.html

```diff
@@ -148,6 +151,19 @@ <h3 class="collection_stats_container"><span class="collection_stats"></span> &n
 							</ul>
 						</div>
 					</div>
+					
+					<div id="gif" class="tab-pane fade">
+						<div id="gifContent" style="text-align: center">
+							<div class="intervalTab">		
+								<label class="inputTitle">Interval Time</label>
```

Comment:
See my earlier comment about this label.

Reply:
about the indentation, right?

Reply:
No, I suggested a change in the phrase `Interval Time` to make it something like `Animation Interval`. Interval already means a gap/period of time, so adding time after it is redundant. I think my previous suggestion was lost for some reason.

As an aside, if a reviewer suggests some changes in a PR using built-in suggestions mechanism and you plan to accept those changes, it is recommended to incorporate them using appropriate controls to give them credit for the change. This is completely on the discretion of the PR author though. The technique is described at https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request

Reply:
I just looked at that piece of code and I indented them in that way because those divs are inside the other

Reply:
Oh okay, I'm sorry about that

Reply:
I did not complain about the indentation here. Indentation inconsistency is mostly in the JS parts (though, HTMLs can also be improved). Here I meant the change in the `Interval Time` text itself to make it something like `Animation Interval`.

Reply:
Okay, got it! I changed that. Sorry about the inconvenience. I'm new to this platform and you've been extremely helpful. Thank you!

Reply:
It's alright, it's part of the learning process. Thanks for improving this application.